newIDE/app/src/VariablesList/VariablesEditorDialog.js

```diff
@@ -63,14 +62,6 @@ const VariablesEditorDialog = ({
       cannotBeDismissed={true}
       onRequestClose={onCancelChanges}
       secondaryActions={[
```

Comment:
You've removed a feature here, this means we can't open the object variables anymore when we edit the variables of an instance.
Add it back and find a way instead to open the object editor with the variable tab opened :) 

This means you'll have to see where this is used, search for `onEditObjectVariables` in the codebase.
You'll find it in SceneEditor/index.js (ignore the things related to `UseSceneEditorCommands`, it's a totally different component), here precisely:

https://github.com/4ian/GDevelop/blob/2517b4740111a3dc790f27f6c7b4d671b26a0f1d/newIDE/app/src/SceneEditor/index.js#L1315-L1331

If you read this, you see this opens a variable editor for the object. You need to:
1) Replace this by a call to `this.editObject(object, 'variables')` - simple! :) 
2) A bit harder: now you must clean all usage of `this.state.variablesEditedObject` and remove `<VariablesEditorDialog` which is related to `this.state.variablesEditedObject`. Why? Because this dialog is now useless, as we moved the variables edition inside the object editor :)